### ISSUE TITLE
Portable Version

### ISSUE THREAD

**Issue by omodia** [Ambratolm/image-resizer#1]
Can you provide portable version of the app?

**Comment by Ambratolm** [comment]
Thanks for your interest in this. great suggestions. I haven't touched this long time ago, but would be happy to revisit it as soon as I can :)

**Comment by HildaJ** [comment]
I agree with @omodia. A portable version of this app would be great. Thank you for coding this app.
